Scouting/src/main/java/ca/team2706/scouting/mcmergemanager/gui/MainActivity.java

```diff
@@ -229,9 +230,16 @@ public void onWriteCommentButtonClick (View v) {
         teamNumber = enterATeamNumberPopup.getTeamNumber();
         comment = commentTextEditor.getComment();
 
-        // TODO To add saving with commentList
+        CommentList commentList = new CommentList(1114);
```

Comment:
I'm guessing "team: 1114" and "testing comment" was a placeholder from before you made the popups, and you forgot to remove it?

Reply:
Yeah, I just did that because of the new comment bar idea we had for the front end which would require me to scrap all of this anyway.

Reply:
You know, now that I see the popups working, I kinda like it. I could go either way :)

Reply:
Regardless, it's a very easy change.